Added optional attribute for validation report

heuerfin opened the pull request:
# Description

Added optional attribute for a validation report in OSITrace shacl and example.

## Type of change


- [x] Change (non-breaking change or fix on an existing type)

## How Has This Been Tested?

Run github pipeline checks locally and in github actions.

## Checklist

- [x] My code follows the modelling guidelines of this project
- [x] I have performed a self-review of my own changes

pmai reviewed (approved):
Looks good; such an attribute might however make sense for all asset classes (common content quality attribute) going forward.

One might also want to add a mime type attribute giving the type/format of the report (pdf, xqar, ...), so that no content sniffing is needed. And might be a repeatable field (multiple different aspects being validated; and/or different formats). But as said this would be more for the future.

heuerfin commented:
> ood; such an attribute might however make sense for all asset classes (common content quality attribute) going forward.
> 
> One might also want to add a mime type attribute giving the type/format of the report (pdf, xqar, ...), so that no content sniffing is needed. And might be a repeatable fi

I've added a baisc attribute for type. I'm currently not aware of any `xsd` or `shacl` compatible with `mime-type` so i kept it at `xsd:string` and added it in the description. Maybe @robertschubert knows a type for `mime-type`. And since it is fine for you, I'll add now @robertschubert also as reviewer since the technical review is done.
